+recovery/+functionality/fn_tenant_function.m

```diff
@@ -448,9 +448,9 @@
 subsystem_failure = ratio_damaged > subsystem_threshold;
 
 % Calculate tenant unit recovery day for this subsystem
-subsystem_recovery_day = max(subsystem_failure .* repair_complete_day,[],2);
+subsystem_recovery_day = max(subsystem_filter .* subsystem_failure .* repair_complete_day,[],2);
 
 % Distrbute recovery day to the components affecting function for this subsystem
-subsystem_comp_recovery_day = subsystem_filter .* initial_damaged .* subsystem_recovery_day;
```

Comment:
I think the subsystem_filter is still required when distributing the recovery day to the components within the subsystem. Otherwise that recovery day will be distributed to all components that have initial damage (regardless of system)

Reply:
Great catch - made the change now!